### DISCUSSION TITLE
Simple SELECT * (SELECT ALL)

### DISCUSSION THREAD

**Discussion by chikko80** [SeaQL/sea-query#719] [Q&A] [answered]
Hey, 
i must have missed this, but how do you guys do SELECT * with sea query?

I scrolled through the readme on multiple days and still couldn't figure it out, probably a simple one

Happy for any answer

**Comment by tyt2y3** [comment, marked as answer]
There is a symbol `Asterisk` and you use it like a normal Iden `column(Asterisk)`

https://docs.rs/sea-query/latest/sea_query/types/struct.Asterisk.html

**Comment by k3dom** [comment]
I think this should really be included in the README.md as an example. I almost lost my mind trying to figure out how to do literally the most basic SQL query until i stumbled upon this discussion. This is the problem with docs.rs as a documentation. Discoverability is sadly just really bad.
